Java/libraries/recognizers-text-number/src/main/java/com/microsoft/recognizers/text/number/resources/EnglishNumeric.java

```diff
@@ -75,7 +75,7 @@ public static String NumbersWithPlaceHolder(String placeholder) {
 
     public static final String NumberOrdinalRegex = "(first|second|third|fourth|fifth|sixth|seventh|eighth|ninth|tenth|eleventh|twelfth|thirteenth|fourteenth|fifteenth|sixteenth|seventeenth|eighteenth|nineteenth|twentieth|thirtieth|fortieth|fiftieth|sixtieth|seventieth|eightieth|ninetieth)";
 
-    public static final String RelativeOrdinalRegex = "(last|next one|previous one|the second to last|the one before the last one|next to last|penultimate|the last but one|antepenultimate)";
+    public static final String RelativeOrdinalRegex = "(last|next|next one|previous one|the second to last|the one before the last one|next to last|penultimate|the last but one|antepenultimate)";
```

Comment:
This seems strange. Was this file auto-generated? "next one" should come before "next" for it to work correctly. Also, why is the order in the regex only different in Java?